docs/reference/index-config.md

```diff
@@ -1,53 +1,73 @@
 ---
 title: Index configuration
-position: 2
+position: 3
 ---
 
-The index config lets you define four things:
-- the doc mapping, that is how a document, and the fields it contains are stored and indexed for a given index. A document is a collection of named fields, each having its own data type (text, binary, date, i64, f64)
-- the timestamp field `timestamp_field` used for [sharding documents in splits](../overview/architecture.md#splits). This is very useful when querying as Quickwit will be able to prune splits based on time range and make search way faster. The timestamp field must be an `i64`. When you define a timestamp field, note that documents will be ordered by default in descending order related to this field
-- the default search fields `default_search_fields`: if no field name is specified in your query, these fields will be used for search
-- whether or not the original JSON document is stored or not in the index by setting `store_source` to true or false.
+This page describes how to configure an index.
 
-This config can be expressed as a json file given to the `new` Quickwit command. Here is a example of a json config for a logging dataset:
+The index config lets you define four things:
+- The document model version, only version `0` is available.
+- The doc mapping, that is how a document and the fields it contains are stored and indexed for a given index.
+- The indexing settings that defines the merge policy, timestamp field used for sharding, and more.
+- The search settings that defines the default search fields `default_search_fields`, a list of field that will be used by default for search.
+- The (data) sources that defines a list of source of types among Kafka or simple files.  
+
+Configuration is set at index creation or can be modified later by using CLI commands or by directly changing settings in the metastore except for the doc mapping (or schema), which is today immutable.
```

Comment:
I think only sources can be changed later via CLI

Reply:
Yes indeed, I will update that on notion.